### PR TITLE
fix unsafe variance

### PR DESCRIPTION
**Pull request by KotlinIsland**
amongus is unsafe fr fr

- resolves: #353

### CODE REVIEW

File: mypy/errorcodes.py [hunk 1]
```diff
@@ -160,8 +160,10 @@ def __str__(self) -> str:
     "General",
     default_enabled=False,
 )
-UNSAFE_VARIANCE: Final = ErrorCode("unsafe-variance", "Incorrect usages of variance", "General")
-UNUSED_IGNORE: Final = ErrorCode("unused-ignore", "Ignore comment is unused", "General")
+UNSAFE_VARIANCE: Final[ErrorCode] = ErrorCode(
+    "unsafe-variance", "Incorrect usages of variance", "General"
+)
+UNUSED_IGNORE: Final[ErrorCode] = ErrorCode("unused-ignore", "Ignore comment is unused", "General")
```
Comment (DetachHead) [inline mypy/errorcodes.py:166]: why explicitly specify these `Final`s?

Comment (KotlinIsland) [reply mypy/errorcodes.py:166]: There was some weird bug where it couldn't infer it.

Comment (DetachHead) [reply mypy/errorcodes.py:166]: i would add a comment/issue link

Comment (KotlinIsland) [reply mypy/errorcodes.py:166]: #373

**Review by DetachHead — APPROVED** [review]

**Pull request merged**